packages/jellyfish-crypto/src/aes256.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Implementation reference: https://github.com/JamesMGreene/node-aes256
+ */
+import crypto from 'crypto'
+
+const CIPHER_ALGORITHM = 'aes-256-ctr'
+
+/**
+ * Encrypt a clear-text message using AES-256 plus a random Initialization Vector.
+ * @param {Buffer} key  A passphrase of any length to used to generate a symmetric session key.
+ * @param {Buffer} data  The clear-text message or buffer to be encrypted.
+ * @returns {Buffer}
+ */
+function encrypt (key: Buffer, data: Buffer): Buffer {
+  const sha256 = crypto.createHash('sha256')
+  sha256.update(key)
+
+  const iv = crypto.randomBytes(16)
```

Comment:
We need the ability to provide a `randomBytes()` implementation. Using JavaScript crypto.randomBytes is too dangerous in production.

Similar to how it's done in:

https://github.com/DeFiCh/jellyfish/blob/e2d7a6faff74104bd5521d46eff155c13a4545e1/packages/jellyfish-wallet-mnemonic/src/mnemonic.ts#L57-L61

Reply:
I guess, generateMnemonic rng is called multiple times. For this, I think allow custom `iv` buffer arg instead of `rng()` should be sufficient.

Reply:
It's called once, the reason we need to provide is that JavaScript Crypto is not using secured random bytes. You need to inject it from native.